examples/run-blockchain/index.js

```diff
@@ -0,0 +1,172 @@
+const async = require('async')
+const ethUtil = require('ethereumjs-util')
+const Trie = require('merkle-patricia-tree/secure')
+const Block = require('ethereumjs-block')
+const Blockchain = require('ethereumjs-blockchain')
+const BlockHeader = require('ethereumjs-block/header.js')
+const VM = require('../../')
+const Level = require('levelup')
+const Account = require('ethereumjs-account')
+const utils = require('ethereumjs-util')
+const BN = utils.BN
+const rlp = utils.rlp
+const testData = require('./test-data')
+// inMemory blockchainDB
+var blockchainDB = new Level('', { db: require('memdown') })
+
+var state = new Trie()
+
+var blockchain = new Blockchain(blockchainDB)
+blockchain.ethash.cacheDB = new Level('./.cachedb')
```

Comment:
the first time the ethash runs it has to generate a large data set that is stored in here.